pkg/flowkit/util/mixpanel.go

```diff
@@ -31,7 +31,7 @@ import (
 
 const (
 	MIXPANEL_TRACK_URL   = "https://api.mixpanel.com/track"
-	MIXPANEL_QUERY_URL   = "https://mixpanel.com/api/2.0/engage?project_id=2154593"
+	MIXPANEL_QUERY_URL   = "https://mixpanel.com/api/2.0/engage?project_id=2123215"
```

Comment:
how come this changed now? don't we use the same token?

Reply:
We change to use flow production project in mixpanel

Reply:
Ok I see.